src/software/Localization/main_rigLocalizer.cpp

```diff
@@ -249,24 +253,30 @@ int main(int argc, char** argv)
 #endif
                                            );
     param = new localization::VoctreeLocalizer::Parameters();
```

Comment:
@fabiencastan regarding the `vocLocParams` u suggested, can/shall i just do 
```c++
param.reset(new ...)
```
if i use `param` as a `std::unique`?